Added sample for DLP : Create and get job

sameer-crest opened the pull request:
Implemented below samples
[1) dlp_create_job](https://cloud.google.com/dlp/docs/samples/dlp-create-job#dlp_create_job-java)
[2) dlp_get_job](https://cloud.google.com/dlp/docs/samples/dlp-get-job#dlp_get_job-java)

Added test cases for the same

sameer-crest commented:
Hi @bshaffer , Please add this ENV variable  **GCS_PATH** 
Example : 
**GCS_PATH**= gs://GOOGLE_STORAGE_BUCKET/dlp_sample.csv
And upload below file into bucket 
[dlp_sample.csv](https://github.com/GoogleCloudPlatform/php-docs-samples/files/11576800/dlp_sample.csv)

bshaffer commented:
@sameer-crest I added it, but some tests (8.1 and 8.2) were passing without it, which is worrisome. Would you be able to look into that?

sameer-crest commented:
> @sameer-crest I added it, but some tests (8.1 and 8.2) were passing without it, which is worrisome. Would you be able to look into that?

@bshaffer I have removed the default value from the sample.
